src/components/Search/GlobalSearchBar.tsx

```diff
@@ -0,0 +1,55 @@
+import {useState} from 'react';
+import SearchIcon from '@/assets/main/search-icon.svg';
+import ClearIcon from '@/assets/main/clear-icon.svg';
+
+interface ContestSearchBarProps {
+  onSearch: (keyword: string) => void;
+  placeholder: string;
+}
+
+export default function ContestSearchBar({
```

Comment:
파일명은 Global로 변경해 주셨는데 선언부가 변경이 안 되었네요 그리고 컴포넌트 export는 다음과 같은 방식으로 통일하겠습니다
```suggestion
export const GlobalSearchBar = () = {
```